uint32 seqID
uint32 imageHwTimeStamp
float32[] fusionPoints
float32[] range

